sdk/python/setup.py

```diff
@@ -77,7 +77,8 @@ def find_version(*file_path_parts):
     name=NAME,
     version=find_version('kfp', '__init__.py'),
     description='KubeFlow Pipelines SDK',
-    author='google',
+    author='The Kubeflow Authors',
+    url='https://kubeflow-pipelines.readthedocs.io/en/stable/',
```

Comment:
I wonder whether we should link to the repo.

Reply:
```
    url="https://github.com/kubeflow/pipelines",
    project_urls={
        "Documentation": "https://kubeflow-pipelines.readthedocs.io/en/stable/",
        "Bug Tracker": "https://github.com/kubeflow/pipelines/issues",
    },
```

Reply:
done!